src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -1414,15 +1414,29 @@ public object ConvertBack(object value, Type targetType, object parameter, Syste
         }
     }
 
+    //TODO remove(this is not used anywhere) in Dynamo 3.0
```

Comment:
@saintentropy this is the converter I was talking about .... is this the one you were thinking of? I can't find this particular one used anywhere.